src/Integrations/Integrations/Integration.php

```diff
@@ -68,9 +68,9 @@ public function addTraceAnalyticsIfEnabledLegacy(Span $span)
      * Sets common error tags for an exception.
      *
      * @param SpanData $span
-     * @param \Exception $exception
+     * @param \Throwable $exception
      */
-    public function setError(SpanData $span, \Exception $exception)
+    public function setError(SpanData $span, $exception)
```

Comment:
Can we add the `\Throwable` type to the argument also?
```suggestion
    public function setError(SpanData $span, \Throwable $exception)
```

Reply:
No we cannot, not as long as we're still releasing PHP 5 and PHP7+ code in the same release. Extension code is still independent, but PHP code is mostly still shared.